pymfe/_summary.py

```diff
@@ -277,36 +275,36 @@ def sum_nanstd(values: TypeValList, ddof: int = 1) -> float:
     if len(values) <= ddof:
         return np.nan
 
-    return np.nanstd(values, ddof=ddof)
+    return float(np.nanstd(values, ddof=ddof))
```

Comment:
Why is conversion to float now necessary?

Reply:
The reason is that the numpy functions can either return a numpy array (if the input array has more than one dimension and the numpy function is not applied in all dimensions) or a single number (if the input array has a single dimension). Therefore, the mypy does not know how to handle that ambiguity.

Here I am just enforcing the data type to help the static code checker, even though the code execution should no be affected by any means in that case.

Reply:
Thanks for the explanation and for updating me on the new version of mypy.